invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/components/DocumentListEntry/DocumentListEntry.js

```diff
@@ -0,0 +1,186 @@
+  if (!edition) return null;
+  return (
+    <Item.Description>
+      edition&nbsp;<strong>{edition}</strong>
+    </Item.Description>
+  );
+};
+
+const getTags = tags => {
+  if (_isEmpty(tags)) return null;
+  return (
+    <Item.Meta>
+      {tags.map(tag => (
+        <Label
+          as={Link}
+          className={'highlighted'}
+          key={tag.pid}
+          size={'small'}
+          content={tag.name}
+          to={BackOfficeRoutes.documentsListWithQuery(
+            '&sort=mostrecent&order=desc&aggr[0][tags][value]=tags.' + tag.name
+          )}
+        />
+      ))}
+    </Item.Meta>
+  );
+};
+
+const getOverbookIcon = isOverbooked => {
+  const name = isOverbooked ? 'check' : 'minus';
+  const color = isOverbooked ? 'green' : 'grey';
+  return <Icon name={name} color={color} size="small" style={{ margin: 0 }} />;
+};
+
+const getStats = meta => {
+  return (
+    <List verticalAlign="middle" style={{ color: '#aaa' }}>
+      <List.Item>
+        <List.Content floated="right">
+          <Header as={Link} to={meta.loan_count_url}>
+            {meta.loan_count}
+          </Header>
+        </List.Content>
+        <List.Content>
+          <Header as="h4">total loans</Header>
+        </List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Content floated="right">
+          <Header as="h4">{meta.loan_extensions}</Header>
+        </List.Content>
+        <List.Content>
+          <Header as="h4">total extensions</Header>
+        </List.Content>
+      </List.Item>
+    </List>
+  );
+};
+
+const getCirculation = circ => {
+  return (
+    <List verticalAlign="middle" style={{ color: '#aaa' }}>
+      <List.Item>
+        <List.Content floated="right">
+          <strong>{circ.has_items_for_loan}</strong>
+        </List.Content>
+        <List.Content>available items</List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Content floated="right">
+          <strong>{circ.active_loans}</strong>
+        </List.Content>
+        <List.Content>active loans</List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Content floated="right">
+          <strong>{circ.pending_loans}</strong>
+        </List.Content>
+        <List.Content>pending loans</List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Content floated="right">
+          {getOverbookIcon(circ.overbooked)}
+        </List.Content>
+        <List.Content>overbooked</List.Content>
+      </List.Item>
+    </List>
+  );
+};
+
+const getExtra = document =>
+  document.metadata.loan_count
+    ? getStats(document.metadata)
+    : getCirculation(document.metadata.circulation);
+
+export function DocumentListEntry({ document }) {
+  return (
+    <Grid.Row columns={2}>
+      <Grid.Column width={3} textAlign={'center'}>
+        <Image
+          as={Link}
+          to={BackOfficeRoutes.documentDetailsFor(document.metadata.pid)}
+          size={CARD_IMAGE_SIZE}
+          src={getCover(document.metadata.pid)}
```

Comment:
Personally I would be more in favour of naming the functions that return components without the get part. E.g `getRestrictions` -> `Restrictions`. So in that fashion the `getCover` and `getRestrictions` would be clear for me if they are just functions or presentational components. WDYT?